ruby_on_rails/depfu.md

```diff
@@ -10,3 +10,49 @@ repository.
 That's all :-)
 
 Update strategy should be set to `Grouped Updates`, frequency: `monthly`. Assignee should be set.
+
+## Engine Updates
+
+Enable minor engine updates.
+
+![Depfu Engine Updates](../images/depfu_engine_updates.png)
+
+**Note:** If you are using Heroku, the latest Ruby / node version may not yet
+be available on their platform, so you may need to delay the upgrade. Check the
+following GitHub repositories to see if Heroku added support already:
+
+* <https://github.com/heroku/heroku-buildpack-nodejs>
+* <https://github.com/heroku/heroku-buildpack-ruby>
+
+## Automatic merging
+
+To speed up the merging of smaller upgrades (like security fixes) we enable
+Automatic merging like this:
+
+![Depfu Automatic Merging](../images/depfu_automatic_merging.png)
+
+Since PRs need to be approved before depfu can merge them we add a GitHub
+Actions workflow to automatically approve PRs from depfu:
+
+```yaml
+# .github/workflows/depfu_autoapprove.yml
+name: Depfu auto-approve
+on:
+  pull_request_target:
+    types: [opened]
+
+permissions:
+  contents: write
```

Comment:
Is the `contents: write` permission actually needed? A quick look at the table at https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs suggests that the pull-requests permission might be the only requirement

Reply:
Good point. I'm not sure how I ended up with specifying both originally.

Reply:
Tested it and it works. Fixed.